api/src/test/java/com/brandwatch/ahmedanalytics/api/integrationtests/utility/DBManager.java

```diff
@@ -0,0 +1,39 @@
+package com.brandwatch.ahmedanalytics.api.integrationtests.utility;
+
+
+import com.brandwatch.ahmedanalytics.common.entities.Mention;
+import com.brandwatch.ahmedanalytics.common.entities.Query;
+import com.brandwatch.ahmedanalytics.common.repositories.MentionsRepository;
+import com.brandwatch.ahmedanalytics.common.repositories.QueryRepository;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class DBManager {
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    @Autowired
+    private QueryRepository queryRepository;
+
+    @Autowired
+    private MentionsRepository mentionsRepository;
+
+    public void populateDB() throws IOException {
+        List<Query> queries = mapper.readValue(new File("src/test/java/com/brandwatch/ahmedanalytics/api/integrationtests/utility/querytestdata.json"), new TypeReference<List<Query>>(){});
```

Comment:
I would expect these in a `resources` folder I think 🤔

Reply:
Yeah, I moved these 2 JSON file to a new folder in resources after I made the PR and forgot to push that :-)